R/dal.R

```diff
@@ -164,8 +170,8 @@ edav_io <- function(
   }
 
   if (io == "write") {
-    if (!grepl(".rds|.csv", file_loc)) {
-      stop("At the moment only 'rds' 'rda' and 'csv' are supported for reading.")
+    if (!grepl(".rds|.csv|.png", file_loc)) {
```

Comment:
What do you think about leaving this open and instead of being ".png" specific we allow the writing of any file time using this method? That way we can just ask people to provide a path to a local file and if it's not a csv/rds etc it will be loaded using the name that people gave it locally?